client/src/pages/EditProfile.js

```diff
@@ -60,6 +76,8 @@ class EditProfilePage extends Component {
   handleSubmit = event => {
     event.preventDefault();
     console.log(this.state);
+    this.setState({ edit: "0" });
+
     this.setState({ disabled: true });
   };
```

Comment:
This too. We had axios set up in handleSubmit. You probably didn't pull latest changes before working on this file.